cargo-frc/src/deploy.rs

```diff
@@ -55,6 +55,13 @@ fn make_addresses(team_number: u64) -> Vec<String> {
         format!("roborio-{}-FRC.local", team_number),
         format!("10.{}.{}.2", team_number / 100, team_number % 100),
         "172.22.11.2".to_string(),
+
+        // The following are addresses that exist in uncommon development environments
+        // Some teams run a shop wifi network that acts as a practice field. 
+        // This setup involves the following mDNS entries
+        format!("roborio-{}-FRC", team_number),
+        format!("roborio-{}-FRC.lan", team_number),
```

Comment:
Technically these aren't mDNS, but LGTM :+1:

Reply:
True. `.lan` is a loose standard used by some routers, and the first one, I have no idea what it is, but it works. Hostname lookup maybe?

Reply:
Yeah these are just plain DNS.